enclave/include/enclave_context.h

```diff
@@ -319,13 +365,18 @@ class EnclaveContext {
       unsigned char * nameptr = name.p;
       size_t name_len = name.len;
 
-      //fprintf(stdout, "Decrypted key\n");
-      //for (int i = 0; i < CIPHER_KEY_SIZE; i++)
-      //  fprintf(stdout, "%d\t", output[i]);
-      //fprintf(stdout, "\n");
-      std::vector<uint8_t> v(output, output + CIPHER_KEY_SIZE);
+      // storing user private key
+      std::vector<uint8_t> user_private_key(output, output + CIPHER_KEY_SIZE);
       std::string user_nam =  convertToString((char *)nameptr, name_len);
-      client_keys.insert({user_nam, v});
+      client_keys.insert({user_nam, user_private_key});
+
+      // storing user public key
+      // TODO verify that user certificate's public key has the same length as the secure enclave public key
+      mbedtls_rsa_alt_context * rsa_info = (mbedtls_rsa_alt_context *) &(user_cert.pk.pk_info);
```

Comment:
To stay consistent with formatting, let's either put the `*` together with the type or the variable name, i.e. `mbedtls_rsa_alt_context* rsa_info` or `mbedtls_rsa_alt_context *rsa_info`.

Reply:
fixed